Source/iOS/AppLovin-MAX-Godot-Plugin/AppLovinMAXGodotPlugin.mm

```diff
@@ -355,6 +382,113 @@ void applovin_max_godot_plugin_deinit()
     return !ALUtils.simulator;
 }
 
+#pragma mark - Consent Flow
+
+void AppLovinMAXGodotPlugin::set_terms_and_privacy_policy_flow_enabled(bool enabled)
+{
+    _sdk.settings.termsAndPrivacyPolicyFlowSettings.enabled = enabled;
+}
+
+void AppLovinMAXGodotPlugin::set_privacy_policy_url(String url_string)
+{
+    _sdk.settings.termsAndPrivacyPolicyFlowSettings.privacyPolicyURL = [NSURL URLWithString: NSSTRING(url_string)];
+}
+
+void AppLovinMAXGodotPlugin::set_terms_of_service_url(String url_string)
+{
+    _sdk.settings.termsAndPrivacyPolicyFlowSettings.termsOfServiceURL = [NSURL URLWithString: NSSTRING(url_string)];
+}
+
+void AppLovinMAXGodotPlugin::set_consent_flow_debug_user_geography(String user_geography)
+{
+    _sdk.settings.termsAndPrivacyPolicyFlowSettings.debugUserGeography = (ALConsentFlowUserGeography) NSSTRING(user_geography).intValue;
+}
+
+void AppLovinMAXGodotPlugin::show_cmp_for_existing_user()
+{
+    if ( !_isSdkInitialized )
+    {
+        NSLog(@"[%@] Failed to show CMP for existing user - please ensure the AppLovin MAX Godot Plugin has been initialized by calling 'AppLovinMAX.initialize()'!", TAG);
+        return;
+    }
+    
+    [_sdk.cmpService showCMPForExistingUserWithCompletion:^(ALCMPError * _Nullable error) {
+        
+        Dictionary cmp_error = Dictionary();
+
+        if ( error )
+        {
+            cmp_error["code"] = (int) error.code;
+            cmp_error["message"] = GODOT_STRING(error.message);
+            cmp_error["cmpCode"] =  (int) error.cmpCode;
+            cmp_error["cmpMessage"] = GODOT_STRING(error.cmpMessage);
+        }
+
+        call_deferred("emit_signal", AppLovinMAXSignalCmpOnShowForExistingUser, cmp_error);
```

Comment:
I feel `cmp_error` should be `NULL` if there is no error right? b/c I think how pubs integrate will just be a null check on `cmp_error` without inspecting if it actually has valid contents.

Reply:
`NULL` can't be emitted with a signal, but the Godot side returns `null` when the returned dictionary is empty.